engine/time/src/main/java/io/deephaven/time/DateTimeUtils.java

```diff
@@ -2600,11 +2751,27 @@ public static int hourOfDay(@Nullable final ZonedDateTime dateTime) {
         return minuteOfDay(dateTime) / 60;
     }
 
+    /**
+     * Returns a 1-based int value of the day of the week for a {@link ZonedDateTime} in the specified time zone, with 1
+     * being Monday and 7 being Sunday.
+     *
+     * @param date date to find the day of the week of
+     * @return {@link QueryConstants#NULL_INT} if either input is {@code null}; otherwise, the day of the week
+     */
+    @ScriptApi
+    public static int dayOfWeek(@Nullable final LocalDate date) {
```

Comment:
Should we have `DayOfWeek dayOfWeek(...)` and `int dayOfWeekValue(...)`?

Reply:
I do like this change, but it is breaking.  I'll poll for concerns.